src/sentry/models/useroption.py

```diff
@@ -21,6 +21,24 @@ class UserOptionValue(object):
     # 'workflow:notifications'
     all_conversations = '0'
     participating_only = '1'
+    # 'deploy-emails
+    all_deploys = '2'
+    committed_deploys_only = '3'
+    no_deploys = '4'
+
+option_scope_error = NotImplementedError('this is not a supported use case, scope to project OR organization')
```

Comment:
so, my fault for not being clearer, but i meant you could put the message in a variable and do something like `raise NotImplementedError(option_scope_error)` when you're using it. you could also subclass `NotImplementedError`.